Settings.c

```diff
@@ -137,10 +137,18 @@ static bool Settings_read(Settings* this, const char* fileName, int initialCpuCo
       } else if (String_eq(option[0], "sort_key")) {
          // This "+1" is for compatibility with the older enum format.
          this->sortKey = atoi(option[1]) + 1;
+      } else if (String_eq(option[0], "tree_sort_key")) {
+         // This "+1" is for compatibility with the older enum format.
+         this->treeSortKey = atoi(option[1]) + 1;
       } else if (String_eq(option[0], "sort_direction")) {
          this->direction = atoi(option[1]);
+      } else if (String_eq(option[0], "tree_sort_direction")) {
+         // This "+1" is for compatibility with the older enum format.
+         this->treeDirection = atoi(option[1]) + 1;
```

Comment:
Shouldn't this align with the storage of `this->direction`?
```suggestion
         this->treeDirection = atoi(option[1]);
```

Reply:
Oops, copy-paste error! Fixed!